docs/topic/platform.upload.announce.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+---
+\[[Front page](../overall-architecture.html)\] \[[Overall architecture](../overall-architecture.html)\]
+
+
+
+# Topic: `patform.upload.announce`
+
+
+
+## Topic description
+
+The records (messages) received from `platform.upload.announce` that are
+identified by header. Messages to be consumed by CCX Data Pipeline contain
+header `openshift` and are encoded using JSON format. Such messages are sent
```

Comment:
The header key is `service`, and the value for the messages that we want to take is `openshift`

Reply:
fixed